### PR TITLE
Add nmtui & nmcli prompt to 'myip' output when no IP address found #96

### LINKED ISSUE #96: Empty output from `myip` when network connection is unavailable
When the network connection is unavailable, the `myip` script does not return anything, which is not the most explicit. Maybe we could improve the user feedback by letting the user know that there was no IP to be returned, which is indicative of a network issue. See screenshot below:
image

A comment in `myip` does indicate the desire to improve that script so addressing this issue could be done then.

PS: @phillxnet , note that this was seen in openQA ;)

### PR DESCRIPTION
**Pull request by phillxnet**
Usability enhancement to prompt folks to use NetworkManager command line tools when our 'myip' helper finds no IP address.

Fixes #96 
Ready for review.

## Testing
Edited a local installer instance of myip as per proposed change and tested as per issue text:

myip-noip-fix

### CODE REVIEW

**Comment by phillxnet** [comment]
Linking to a related, in part, documentation issue regarding static ip setup:
https://github.com/rockstor/rockstor-doc/issues/358

File: root/root/bin/myip [hunk 1]
```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
-# Quick hack to get ip address, change to wrapper around python script.
-ip a | grep -oE '((1?[0-9][0-9]?|2[0-4][0-9]|25[0-5])\.){3}(1?[0-9][0-9]?|2[0-4][0-9]|25[0-5])' | grep -v '127\|255' | sed -e "s/^/https:\/\//"
+# Quick hack to get ip address.
+MYIP=$(ip a | grep -oE '((1?[0-9][0-9]?|2[0-4][0-9]|25[0-5])\.){3}(1?[0-9][0-9]?|2[0-4][0-9]|25[0-5])' | grep -v '127\|255' | sed -e "s/^/https:\/\//")
+if [[ $MYIP ]]
+then
+    echo $MYIP
+else
+    echo "No DHCP assigned IP address found."
```
Comment (FroggyFlox) [inline root/root/bin/myip:8]: Does the following read better to you?
```suggestion
    echo "No DHCP-assigned IP address found."
```

**Review by FroggyFlox — COMMENTED** [review]
@phillxnet ,
Thanks for taking care of that one!
I only have minor style suggestion in case it reads better to you.

File: root/root/bin/myip (hunk 1 above)
Comment (phillxnet) [reply root/root/bin/myip:8]: Thanks for the review. Yes this is better, I'll commit.

**Comment by phillxnet** [comment]
## Final testing
An installer was build with this patch pre-applied. The VM enacted an ethernet cable unplugged scenario (see link-status below), to effect no dhcp available via no ethernet connection. An install was then carried out, effectively with-out the ethernet cable plugged in.

The 'myip' utility reacted as intended:
myip-noip-fix_installer-test

nmtui was then used to
1. Configure a static ip
2. enable this connection.
The VM was then instructed to re-connect, live, the ethernet cable via:

link-status-VM-setting

And the consequent  "nmcli" output indicated a transition to a wired carrier:
```
nmcli device show | grep WIRED.PROPERTIES.CARRIER
WIRED-PROPERTIES.CARRIER:               on
```

Thus the resulting static IP was applied to the newly live (freshly plugged in) connection and the Rockstor Web-UI was accessible via the static IP set via nmtui; including after a reboot.

## Caveat
Post our static ip test above we have a not directly related failure in console messaging on future boot messages:

web-ui_is_accessible-messaging-issue

This can and should be addressed in it's own issue and relates to rockstor-core code that re-writes our console message.

**Comment by phillxnet** [comment]
@FroggyFlox Given the above functional end-to-end test I think we are good on this one. Thanks again for the review and I'll merge directly.

**Pull request merged**